core/cleaner/src/main/java/com/cognifide/aet/cleaner/processors/filters/DBKeyProjectCompanyPredicate.java

```diff
@@ -35,6 +35,11 @@ public boolean apply(DBKey dbKey) {
     return companyMatches(dbKey) && projectMatches(dbKey);
   }
 
+  @Override
```

Comment:
WE could use default implementation from the `Predicate` interface providing we change the JAVA level to 1.8 in root pom XML file: https://github.com/Cognifide/aet/commit/dbecf1798778defcf8cf995db6f758f109823eca

What do you think?

Reply:
good point, fixed